packages/@react-types/tooltip/src/index.d.ts

```diff
@@ -30,13 +31,12 @@ export interface AriaTooltipProps extends TooltipProps, DOMProps, AriaLabelingPr
 
 export interface SpectrumTooltipProps extends AriaTooltipProps, StyleProps {
   variant?: 'neutral' | 'positive' | 'negative' | 'info',
-  placement?: 'right' | 'left' | 'top' | 'bottom'
+  placement?: 'right' | 'left' | 'top' | 'bottom',
+  displayIcon?: boolean // is this what we want to name it?
```

Comment:
maybe `displayVariantIcon`?

Reply:
such a loooooong name =P

Reply:
Should be `showIcon` according to dropbox

Reply:
reminder ^